package.json

```diff
@@ -27,23 +27,27 @@
     "intersection-observer": "0.12.0",
     "portal-vue": "2.1.7",
     "vue-i18n": "^8.28.2",
-    "vue-virtual-scroller": "^1.1.2",
-    "webpack-theme-resolver-plugin": "3.0.0"
```

Comment:
Can we delete the info about webpack-theme-resolver-plugin in the README as well?
https://github.com/dobromir-hristov/swift-docc-render/blob/dhristov/update-vue-cli/NOTICE.txt#L78